orm-rules-tests/rules-test/custom-globals/max-timeout-server-0/globals.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+schema_version: 1
+
+crypto:
+  certificates:
+    - /etc/ssl/orm.pem
+
+haproxy:
+  max_server_timeout: 0
```

Comment:
Is it possible to rename the folder and add a comment for this line describing what is done? For example folder name `max-timeout-server-infinite` and add some comment `# Infinite timeout`. Even though I know what is done in this issue I was initially confused by why `max_server_timeout` was zero and why `timeout_server_default` was higher than `max_server_timeout`. It will only be more confusing some weeks later.

Reply:
Sure I'll do that!

It's also possible that actually setting the haproxy defaults -> timeout_server to 0 equals omitting the setting as we do now. In that case, if we instead do that, there is less magic happening in the template. But I guess we will however still use our hardcoded default of 15s when omitting our max_server_timeout (soon timeout_server_max). And in any case we document what it means and translates into.

By using descriptive names instead of key-value based will scale better too, as long as we can come up with good names for everything. That's where I reverted back to key-value based names on the folders. But I can ask for help on that if needed going forward :-)